src/website/views/components/expansion-panel/_examples.pug

```diff
@@ -1462,11 +1471,38 @@ p.-text
           </div>
         </div>
       </div>
-
+      <div class="chi-epanel" data-chi-epanel-group="example__CustomMode">
+        <div class="chi-epanel__header">
+          <div class="chi-epanel__title" data-chi-epanel-action="done">Panel 3</div>
+          <div class="chi-epanel__content">
+            <div class="chi-epanel__collapse">
+              <div class="-done--only">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam varius arcu nisl, non accumsan elit
+                interdum et. Nunc ut gravida justo. Nulla sit amet est accumsan, condimentum elit nec, dapibus nulla.
+                Aenean eu sapien eget ante placerat pretium a sit amet ante.
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="chi-epanel" data-chi-epanel-group="example__CustomMode">
+        <div class="chi-epanel__header">
+          <div class="chi-epanel__title" data-chi-epanel-action="done">Panel 5</div>
+          <div class="chi-epanel__content">
+            <div class="chi-epanel__collapse">
+              <div class="-done--only">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam varius arcu nisl, non accumsan elit
+                interdum et. Nunc ut gravida justo. Nulla sit amet est accumsan, condimentum elit nec, dapibus nulla.
+                Aenean eu sapien eget ante placerat pretium a sit amet ante.
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
       <!-- JavaScript -->
       <script>
         chi.expansionPanel(
-          document.querySelectorAll('[data-chi-epanel-group="example6"]'),
+          document.querySelectorAll('[data-chi-epanel-group="example__CustomMode"]'),
```

Comment:
The same pattern should be followed as for other components using **kebab-case** instead of **CamelCase**: `example__name_subname`. 

```suggestion
          document.querySelectorAll('[data-chi-epanel-group="example__custom_mode"]'),
```

Please check all the IDs.